kratos/containers/model.cpp

```diff
@@ -117,7 +116,7 @@ ModelPart& Model::GetModelPart(const std::string& rFullModelPartName)
     KRATOS_ERROR_IF( rFullModelPartName.empty() ) << "Attempting to find a "
         << "ModelPart with empty name (\"\")!" << std::endl;
 
-    const auto delim_pos = rFullModelPartName.find('.');
+    const auto delim_pos = rFullModelPartName.find(".");
     const std::string& root_model_part_name = rFullModelPartName.substr(0, delim_pos);
```

Comment:
@loumalouomega why did you change this?

Reply:
We use "" for the strings instead of ''

Reply:
aren't those two different things?
'' is a single char and "" is a string no?
@roigcarlo do you know better? :D

Reply:
I think, there are the same, but we use " for strings

Reply:
AFAIK, `'*'` is a char and `"*"` is a string, but i have no idea whether they have any overhead :D

Reply:
Okay, I assume the method find will use a string anyway, and basically does  a cast, so no idea